pom.xml

```diff
@@ -89,13 +89,14 @@
         <jaxws-version>2.3.1</jaxws-version>
 
         <!-- Logging & commons (aligned) -->
-        <log4j-version>2.19.0</log4j-version>
+        <log4j.version>2.19.0</log4j.version>
+        <slf4j.version>1.7.25</slf4j.version>
```

Comment:
Can you please briefly report this in PR description with a more coherent PR title? Thanks.